castepinput/inputs.py

```diff
@@ -55,7 +55,7 @@ def get_file_lines(self):
                 # If a list/tuple is passed join into a string
                 if isinstance(value, (tuple, list)):
                     value = " ".join(map(str, value))
-                if value or value is False:
+                if value is not None and value is not "":
```

Comment:
This should be `if value is not None and value != ""`. `is` compares object id and you are relying on python internally to not create two empty string objects..